packages/components/src/components/dropdown/dropdown.stories.ts

```diff
@@ -101,260 +116,141 @@ export const Default = {
+ * Use the `open` attribute to toggle the visibility of the dropdown.
+ */
+
+export const Open = {
+  name: 'Open',
+  render: () => html`
+    <sd-dropdown open>
+      <div class="slot slot--border slot--text p-4 w-[300px]">Default Slot</div>
+      <sd-button slot="trigger" variant="secondary">Trigger</sd-button>
+    </sd-dropdown>
+  `
+};
+
+/**
+ * Use the `rounded` attribute to set a border-radius on the dropdown panel and trigger button.
  */
 
 export const Rounded = {
-  parameters: { controls: { exclude: ['rounded', 'default'] } },
-  render: (args: any) =>
-    generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'rounded'
-        }
-      },
-      constants: {
-        type: 'slot',
-        name: 'default',
-        value: '<div class="example"></div>'
-      },
-      args
-    })
+  name: 'Rounded',
+  render: () => html`
+    <sd-dropdown open rounded distance="4">
+      <div class="slot slot--border slot--text p-4 w-[300px]">Default Slot</div>
+      <sd-button slot="trigger" variant="secondary">
+        <sd-icon name="system/more-functions" class="h-6 w-6"></sd-icon>
+      </sd-button>
+    </sd-dropdown>
+  `
 };
 
 /**
- * The dropdown automatically adjusts its size to fit the screen. The content gets scrollable then. To disable this behavior, set the `no-auto-size` attribute.
- * Note: `no-flip` is activated here.
+ * Use the `placement` attribute to set the position of the dropdown panel relative to the trigger.
  */
 
-export const NoAutoSize = {
-  parameters: { controls: { exclude: ['default', 'no-flip', 'no-auto-size'] } },
-  render: (args: any) =>
-    generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'no-auto-size'
-        }
-      },
-      constants: [
-        {
-          type: 'slot',
-          name: 'default',
-          value:
-            '<div style="width: 120px;"><div style="height: 110vh; padding: 12px;"><div  class="slot slot--border slot--text slot--background" style="height: 100%; width: auto;">Scroll down<br><br>⬇️</div></div></div>'
-        },
-        {
-          type: 'attribute',
-          name: 'no-flip',
-          value: true
-        }
-      ],
-      args
-    })
+export const Placement = {
+  name: 'Placement',
+  render: () => html`
+    <div class="grid grid-cols-2 gap-32">
+      <sd-dropdown open placement="bottom-start">
+        <div class="slot slot--border slot--text p-4 w-[300px]">Default slot</div>
+        <sd-button slot="trigger">Bottom start</sd-button>
+      </sd-dropdown>
+
+      <sd-dropdown open placement="bottom-end">
+        <div class="slot slot--border slot--text p-4 w-[300px]">Default slot</div>
+        <sd-button slot="trigger">Bottom End</sd-button>
+      </sd-dropdown>
+    </div>
```

Comment:
Please align with design:
Naming: top-start, top-end
Background: neutral-200

![image](https://github.com/user-attachments/assets/6df7c482-e3d6-40e9-818c-5028091bdc95)

Reply:
Design updated to mirror storybook